### PR TITLE
feature/BA-969 

### PR DESCRIPTION
**Pull request by mark-a-wilson**
Bootstrap 4 Styling - Update Core Module (Stragglers)

Fixes BA-969

### CODE REVIEW

File: modules/core/client/views/400.client.view.html [hunk 1]
```diff
@@ -1,9 +1,12 @@
-<div class="container">
-  <h1>Bad Request</h1>
-	<div class="alert alert-danger" role="alert">
-	  <span class="fas fa-exclamation-sign" aria-hidden="true"></span>
-	  <span class="sr-only">Error:</span>
-	  <span ng-if="vm.errorMessage" ng-bind="vm.errorMessage"></span>
```
[the remark this answers is not in the document]
Comment (mark-a-wilson) [reply modules/core/client/views/400.client.view.html]: My intent was to replace what was appearing with custom wording. Is there something in the `vm.errorMessage` that should display in addition to the plain language I composed?

File: modules/core/client/views/404.client.view.html [hunk 2]
```diff
@@ -1,8 +1,12 @@
-<div class="container">
-  	<h1>Page Not Found!</h1>
-	<div class="alert alert-danger" role="alert">
-	  	<span class="fas fa-exclamation-sign" aria-hidden="true"></span>
-	  	<span ng-if="vm.errorMessage" ng-bind="vm.errorMessage"></span>
```
[the remark this answers is not in the document]
Comment (mark-a-wilson) [reply modules/core/client/views/404.client.view.html]: Same comment as above. Apologies if I've misunderstood how this works / should work.

File: modules/core/client/views/400.client.view.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (SteveChapmanBCDX) [reply modules/core/client/views/400.client.view.html]: If you want to restyle the view, you can do that later.  The goal here is to ensure that the existing views can work with Bootstrap 4.

File: modules/core/client/views/404.client.view.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (SteveChapmanBCDX) [reply modules/core/client/views/404.client.view.html]: Same comment as above, which is general for all code changes during the Bootstrap 4 upgrade.

**Comment by SteveChapmanBCDX** [comment]
Commits squashed and merged into dev.

**Pull request closed without merging**